src/Components/Home/Users.js

```diff
@@ -36,7 +38,20 @@ function Users({ list }) {
           filteredList.length > 0 &&
           filteredList.map((user, key) => (
             <Link to={user.username} key={`avatar-${key}`}>
-              <Chip image={user.avatar} className="p-m-2" label={user.name} />
+              <Chip
+                image={user.avatar}
+                className="p-m-2"
+                label={user.name}
+                onImageError={(error) => {
+                  const defaultSVG = createAvatar(style, {
+                    seed: user.name,
+                    dataUri: true,
+                  })
+
```

Comment:
```suggestion

```

Reply:
I do not think this line needs to be removed (it just makes easier to read)